### ISSUE TITLE
automatically detecting binarizer vs scalar

### ISSUE THREAD

**Issue by bbalin12** [scikit-learn-contrib/sklearn-pandas#10]
Question -- I'm dealing with a dataframe of roughly 50 columns of data, where 30 are multi-category categorical string variables.  Is there a way to automatically designate whether a column should have the binarizer applied to it (ie the 30 multi-category variables), and which should be scaled (ie the continuous numeric variables)? 

Using your example above, a way that 

`mapper2 = DataFrameMapper([(['pet', 'salary'], sklearn.decomposition.PCA(1))])`

which mixes string labels and numeric, to work.
